.gitignore

```diff
@@ -426,3 +426,7 @@ tests/integration/ui/cypress/screenshots/
 
 #JetBrains IDE
 .idea/
+package.json
```

Comment:
Why are these excluded and multiple times?

Reply:
it was added by frtibble, I removed it